django/applications/catmaid/management/commands/catmaid_export_data.py

```diff
@@ -2,19 +2,20 @@
 
 from datetime import datetime
 from itertools import chain
-from django.db import connection
-from django.core import serializers
-from django.core.management.base import BaseCommand, CommandError
+from typing import Dict, List, Optional, Set
+
 from catmaid.control.annotation import (get_annotated_entities,
-        get_annotation_to_id_map)
-from django.contrib.auth.hashers import make_password
-from catmaid.control.annotation import get_sub_annotation_ids
+        get_annotation_to_id_map, get_sub_annotation_ids)
```

Comment:
benefits of alphabetical order - redundancy is easy to spot

Reply:
:+1: